internal/sockopt/sockopt_windows.go

```diff
@@ -0,0 +1,9 @@
+//go:build windows
```

Comment:
`plan9` also doesn’t have `SO_ERROR`, or `GetSockoptInt`. (Though, in this case, I don’t think we support it, as it already doesn’t build because `go-metrics` tries to use a non-existent `syslog.Writer`.)

Also, neither `js/wasm` nor `wasip1/wasm` work. These ones at least seem to compile prior to this change.

You may want:
```suggestion
//go:build windows || wasm
```

Though, this will require renaming the file, as the `_windows.go` suffix means this build tag for `windows` is currently redundant, and unnecessary.